### DISCUSSION TITLE
Why range slider doesnt react

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#3947] [Q&A] [answered]
someone could help me? slider doesnt react on change.
[glitch demo](https://mv-playing.glitch.me/)

`  let slider = document.getElementById("animRange");
 slider.max = duration;` 

`  let updateSlider = () => {
   let time = modelViewer.currentTime;
    slider.value = time;
  };` 
 `  slider.addEventListener('input', (event) => {
        let time = modelViewer.currentTime;
    time = event.target.value;
});` 
  
`   self.setInterval(() => {
    updateSlider();
  }, 100);`

**Comment by elalish** [comment, marked as answer]
try `modelViewer.currentTime = event.target.value;`

**Reply by phovorka** [reply]
works!! thanks
